ethdk/src/interfaces/Account.ts

```diff
@@ -0,0 +1,4 @@
+export default interface Account {
+  address: string
+  sendTransaction: (params: any) => Promise<string>
```

Comment:
Should we enforce a type here? Or was the idea to leave it as `any` for now?

Reply:
Yeah good spot, I missed this 👍